interface/code_systems/icd10_howto.php

```diff
@@ -34,7 +34,7 @@
     <p>
         <?php echo xlt("Steps to install the ICD 10 database"); ?>:
         <ol>
-            <li><?php echo xlt("The raw data feed release can be obtained from"); ?> <span class='font-weight-bold'><a href="https://www.cms.gov/Medicare/Coding/ICD10/index" target="_blank" rel="noopener"><?php echo xlt("this location"); ?></a></span>
```

Comment:
I'm not sure to remove span, 
Maybe @bradymiller,  have other good ideas.

Reply:
I kinda hate `span` just before `anchor` because adding class to them helps in modifying the text inside the `anchor` tag but it fails to manipulate the other properties of `anchor` tag. So, in future to change `anchor` properties you will need to add classes/styles to the anchor tag when we already have some class on span.

In short it's my personal opinion :grimacing:

Reply:
As long as there is a rational reason, then sounds good to me.